index.md

```diff
@@ -19,4 +19,4 @@ The Hosting Team meets in the WordPress Slack, in the [#hosting-community](https
 The Hosting Team is represented by team reps [@amykamala](https://profiles.wordpress.org/amykamala/), [@mikeschroder](https://profiles.wordpress.org/mikeschroder/), and [@jadonn](https://profiles.wordpress.org/jadonn/).
 
 
-\[info\]If you’re interested in improving this handbook, check the [Github Handbook repo](https://github.com/WordPress/hosting-handbook/), or leave a message in the [#hosting-community](https://wordpress.slack.com/archives/hosting-community/) channel of the [official WordPress Slack](https://make.wordpress.org/chat/).\[/info\]
+\[info\]If you’re interested in improving this handbook, check the [Github Handbook repo](https://github.com/WordPress/hosting-handbook/), or leave a message in the [#hosting-community channel](https://wordpress.slack.com/archives/hosting-community/) of the official [WordPress Slack](https://make.wordpress.org/chat/).\[/info\]
```

Comment:
Do you still want to make this change to what text is linked after the previous change?

Reply:
yes, I copied the same code for all infoboxes. Last commit was to convert MD to HTML (because it did't worked when we updated everything manually).

Reply:
Okay! It looks like this change is only to where the link is located.

Reply:
(but that's fine of course)